src/components/Profile/Bookmarks.js

```diff
@@ -0,0 +1,128 @@
+// react
+import React, { Component } from 'react';
+
+// 3rd party libraries
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+// styles
+import './Bookmarks.scss';
+
+// components
```

Comment:
nice work @bisonlou on documenting your work, but just not to confuse anyone who would be building on your code, I believe this is a helper function and not a component